### ISSUE TITLE
`UnexpectedText` error when demangling a symbol with a postfix attribute

### ISSUE THREAD

**Issue by koute** [gimli-rs/cpp_demangle#156]
When demangling the symbols of one of our applications the `cpp_demangle` manages to demangle everything besides symbols like these which generate an `Err(UnexpectedText)`:

```rust
let sym = "_ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEE12_M_constructIPcEEvT_S7_St20forward_iterator_tag.isra.90";
println!( "{:?}", cpp_demangle::Symbol::new( sym ) );
```

Using `c++filt` works:
```
void std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_M_construct<char*>(char*, char*, std::forward_iterator_tag) [clone .isra.90]
```

The problem here is the postfix `.isra.90` which should result in `[clone .isra.90]` in the demangled symbol, which isn't supported by `cpp_demangle`.

I see that the libiberty's `test_libiberty_demangle_172` from https://github.com/gimli-rs/cpp_demangle/issues/16 is similar to this, so this is technically a duplicate, but I guess it might be worthwhile to have separate issues to help prioritize the cases which actually pop up in real production binaries?

**Comment by fitzgen** [comment]
Thanks for filing an issue, @koute!

**Comment by miguelsaldivar** [comment]
Seems like `clone-suffix` is missing, I'll add it. :+1:

**Comment by miguelsaldivar** [comment]
This was fixed with #170, forgot to mention this should be closed!

**Comment by fitzgen** [comment]
Thanks!
